server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -64,6 +64,19 @@ def deploy
       end
     end
 
+    def redeploy
+      begin
```

Comment:
@jmrodri No specific reason, just the first idea pitched. `/redeploy` turned out to be much more simple than originally thought.